js/document.js

```diff
@@ -17,6 +17,15 @@ export const DECLARATION_DOC_TYPE = "013";
 
 export const DocumentAPI = superclass =>
     class extends superclass {
+        /**
+         * Uploads a document to the UPS servers.
+         * The uploaded document is returned and its ID can be
+         * used to associate a shipment with the document.
+         *
+         * @param {Object} payload The payload object according to the UPS API standards.

```

Comment:
Is there a link to a document/site that can be added here that would list these standards?

Reply:
This one: https://www.ups.com/upsdeveloperkit?loc=en_US

Should it be @see https://www.ups.com/upsdeveloperkit?loc=en_US ?

Reply:
Yap!

Reply:
Sadly cant get an individual link for each API section. So I will use the generic one for all.